dataflowengine/src/main/scala/io/shiftleft/dataflowengine/passes/reachingdef/ReachingDefPass.scala

```diff
@@ -46,7 +46,7 @@ class ReachingDefPass(cpg: Cpg) extends CpgPass(cpg) {
 
         var inSet = Set[Vertex]()
 
-        val cfgPredecessors = currentCfgNode.vertices(Direction.IN, EdgeTypes.CFG).asScala
+        val cfgPredecessors = currentCfgNode.asInstanceOf[nodes.StoredNode]._cfgIn.asScala
```

Comment:
Instead of all the casting please adjust the type of "currentCfgNode" to StoredNode.